src/main/react-front-end/src/views/Admin/ClientsInfoComponent.js

```diff
@@ -37,6 +44,41 @@ export default class ClientsInfoComponent extends Component{
 		});
 	}
 
+	// Shows the user a confirmation popup to confirm the update request.
+	enableOrDisableAdmin(event, row){
+		var popupMsg = ""
+		var isAdmin = event.target.checked;
+		if(isAdmin){
+			popupMsg = "Do you want to enable "+ row.firstName + " " + row.lastName+ " as admin.";
```

Comment:
Please update the message to
'Please confirm if ______ _______ must be granted admin privileges.'

Reply:
Incorporated